phantomgames/GameDetails/GameDetailViewController.swift

```diff
@@ -4,12 +4,53 @@
 //
 //  Created by Aphiwe Shozi on 2024/05/16.
 //
-#warning("Feature is to be developed in another branch")
-
 import UIKit
 
 class GameDetailViewController: UIViewController {
+    
+    @IBOutlet weak private var gameImageView: UIImageView!
+    @IBOutlet weak private var gameTitleLabel: UILabel!
+    @IBOutlet weak private var gameGenreLabel: UILabel!
+    @IBOutlet weak private var gameDescriptionLabel: UILabel!
+    @IBOutlet weak private var gameReleaseDate: UILabel!
+    @IBOutlet weak private var gamePlatformLabel: UILabel!
+    @IBOutlet weak private var gamePlayButton: UIButton!
+    
+    // MARK:  Variables
+    private lazy var gameDetailViewModel = GameDetailViewModel(repository: GameDetailRepository(), delegate: self)
+    
+    // MARK: Functions
     override func viewDidLoad() {
         super.viewDidLoad()
+        gameDetailViewModel.fetchGameDetail()
+        updateUI()
+    }
+    
+    func assignGameID(gameID: Int) {
+        gameDetailViewModel.updateGameID(gameID: gameID)
+    }
```

Comment:
New line after this function please.
Is there a swift lint warning? Please let me know?
Because either the warning is not showing, or the warning is being ignored?
Please let me know which one it is so I can help. But the purpose of the linter is to catch stuff like this before PR review

Reply:
Ive sent u a slack regarding my linting. The pr for my swift lint is up its pr 17